packages/xstate-svelte/src/useMachine.ts

```diff
@@ -75,5 +106,5 @@ export function useMachine<
     }).unsubscribe;
   });
 
-  return { state, send: service.send, service };
+  return { state, send: service.send, service } as any;
```

Comment:
Wonder why the typings are complaining here?

Reply:
This probably is just missing logic in TypeScript to "resolve" constraints of the conditional types in a generic context (or something like that). Although I've thought that there were some improvements in this area in the latest versions, perhaps they didn't cover a scenario like this

[Minimal repro](https://www.typescriptlang.org/play?#code/C4TwDgpgBAysCGwIB4AqBhA9gOyQD2AD4oBeKAbwCgooBjYPALigx32EoF9LKBLXCACcAZvFrQAsmIAW-FKwEFiVGgHMIwOIgjMAFAEpSxLUjRZFRADTUoAeltRgsgM5Qc0AO6ZBAa2eWoAFt4EAAjaAArAFdnYChw2ngY6CdoAAN4bBA0qHh+AHI4j0F4MEgAE0dpCGwoG3soXmB812coyEEoWJLeVWlgYW8PeEFK4Ew6atofRuEq6EENKMFa0EggxFpq1wysnPhXCAA3IRAnflVG1wPnXux4UIAbFInd7PqHdU0EJD1DEmICnYAS4PDW0AA4hpzOw0MQyKgoBACDVyq4pFs5Mh+MIhFAAErEAD8BKgzGwxyEPGEUWw9F4OCgXxM8gxsgpSJR2DRUDZWMyIEIhF0wUxFOYqD5FP0zChwBhyOAaClEGUNkWwGWtVF7IgADpmT8IAYuEA)